All_home_works/hw6/task2.py

```diff
@@ -106,14 +108,15 @@ def create_homework(text, deadline):
         return Homework(text=text, deadline=deadline)
 
 
-class InstanceNotHomeworkError(Exception):
+class InstanceNotHomeworkError(BaseException):
```

Comment:
Why BaseException?

Reply:
> For educational purposes, let's subclass it from a built-in Exception which suits this case.

And BaseException is the base class for all built-in exceptions.

Reply:
> And BaseException is the base class for all built-in exceptions.

What documentation say about it?

Reply:
`The base class for all built-in exceptions. It is not meant to be directly inherited by user-defined classes (for that, use Exception).`